viewer3d/presets.py

```diff
@@ -469,12 +471,31 @@ def on_invert_change(change):
         style={"description_width": "initial"},
         continuous_update=continuous_update,
     )
+
+    z0_offset = FloatSlider(
+      min=-3,
+      max=3,
+      description="z0_offset",
+      style={"description_width": "initial"},
+      continuous_update=continuous_update,
+    )
+
+    z1_offset = FloatSlider(
+      min=-3,
+      max=3,
+      description="z1_offset",
```

Comment:
Can you also set the `value` attributes in `z0_offset`, `z1_offset` and `delta_omega1`?